.github/workflows/automerge-workflow.yml

```diff
@@ -4,7 +4,8 @@ jobs:
     name: Automerge Workflow
     runs-on: ubuntu-18.04
     steps:
-      - env:
+      - if: env.GITHUB_TOKEN != null
```

Comment:
We check for this and then for Pulumi Bot token below - will that cause a second set of issues if not available?

Reply:
I tested this with https://github.com/pulumi/docs/pull/5981/commits/68eed8278eea7f32e6be00a650b7a747679c51ac and the workflow was skipped, so I'm assuming the workflow will similarly be skipped for runs where GITHUB_TOKEN is absent. If I'm wrong we should be able to revert pretty quickly, though.